lib/config/folderconfiguration.go

```diff
@@ -73,9 +73,13 @@ func (f FolderConfiguration) Path() string {
 	return f.cachedPath
 }
 
+func (f FolderConfiguration) SyncthingPath() string {
+	return filepath.Join(f.Path(), ".syncthing")
+}
+
 func (f *FolderConfiguration) CreateMarker() error {
 	if !f.HasMarker() {
-		marker := filepath.Join(f.Path(), ".stfolder")
+		marker := f.SyncthingPath()
```

Comment:
In the current version we hide .stversions, but I think we should change this indeed.